GirafRest.Test/Controllers/ErrorControllerTest.cs

```diff
@@ -0,0 +1,164 @@
+using Xunit;
+using GirafRest.Controllers;
+using Moq;
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Http.Features;
+
+
+namespace GirafRest.Test.Controllers
+{
+    public class ErrorControllerTest
+    {
+        [Fact]
+       public void Unauthorized401_Test()
+       {
+            // Arrange
+            int status = StatusCodes.Status401Unauthorized;
+            var mockFeatureCollection = new Mock<IFeatureCollection>();
+            mockFeatureCollection.Setup(f => f.Get<IStatusCodeReExecuteFeature>())
```

Comment:
f står for IFeatureCollection ikke? Hvis det gør kan det så ikke få et mere sigende navn så som feature?

Reply:
Jo, det kigger jeg på hurtigest muligt